stable/mongodb/templates/statefulset-secondary-rs.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     matchLabels:
       app: {{ template "mongodb.name" . }}
       release: {{ .Release.Name }}
+      chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
It should be
```
chart: {{ template "mongodb.chart" . }}
```

Reply:
I still see `"{{ .Chart.Name }}-{{ .Chart.Version }}"` instead of `chart: {{ template "mongodb.chart" . }}`